### PR TITLE
Chore/sanitize vat taxid input

### PR DESCRIPTION
**Pull request by saltcod**
## What kind of change does this PR introduce?

Sanitizes the VAT input for EU VAT ids.

### CODE REVIEW

File: studio/components/interfaces/Organization/BillingSettings/TaxID/TaxID.tsx [hunk 1]
```diff
@@ -132,12 +133,33 @@ const TaxID: FC<Props> = ({ loading, taxIds, onTaxIdsUpdated }) => {
     }
   }
 
+  /**
+   * Sanitize EU VAT ids so they get prepended with the country code
+   *
+   * Ex: Belgium's VAT ID should be BE0123456789 not 0123456789
+   * Only EU VAT ids are affected for now
+   */
+  const sanitizeTaxID = (taxId: any) => {
```
Comment (MildTomato) [inline studio/components/interfaces/Organization/BillingSettings/TaxID/TaxID.tsx:142]: shall we get this typed properly ?

File: studio/components/interfaces/Organization/BillingSettings/TaxID/TaxID.tsx [hunk 2]
```diff
@@ -183,16 +205,17 @@ const TaxID: FC<Props> = ({ loading, taxIds, onTaxIdsUpdated }) => {
...
+              <div className="shimmering-loader mx-1 w-2/3 rounded py-3" />
+              <div className="shimmering-loader mx-1 w-1/2 rounded py-3" />
+              <div className="shimmering-loader mx-1 w-1/3 rounded py-3" />
             </div>
           ) : (
-            <Panel.Content className="w-3/5 space-y-4">
+            <Panel.Content className="w-8/12 space-y-4">
               {taxIdValues.length >= 1 ? (
                 <div className="w-full space-y-2">
                   {taxIdValues.map((taxId: any, idx: number) => {
```
Comment (MildTomato) [inline studio/components/interfaces/Organization/BillingSettings/TaxID/TaxID.tsx:216]: use the same type/interface for taxId here

Comment (saltcod) [reply studio/components/interfaces/Organization/BillingSettings/TaxID/TaxID.tsx]: The interface we use in the constants file is different than the one stripe returns, so I created a new one.

File: studio/components/interfaces/Organization/BillingSettings/TaxID/TaxID.tsx (hunk 1 above)
Comment (joshenlim) [reply studio/components/interfaces/Organization/BillingSettings/TaxID/TaxID.tsx:150]: hmm can we shift this out into BillingSettings/TaxID.utils.ts

and then write some tests for it? under tests/components/Billing/TaxID.utils.test.js - give it a couple of test cases using eu tax ids and non eu tax ids

**Review by joshenlim — APPROVED** [review]
Do remember to remove the changes from package-lock.json before we get this in

**Pull request merged**